packages/react-ui/components/Tabs/Tab.tsx

```diff
@@ -224,7 +225,13 @@ export class Tab extends React.Component<TabProps, TabState> {
           ref={isFunctionalComponent(Component) ? null : this.refTabComponent}
           href={href}
         >
-          {children}
+          <ResizeDetector
+            onResize={event => {
+              this.context.notifyUpdate();
+            }}
```

Comment:
Стрелочная функция в пропе - не очень хорошо.

```suggestion
            onResize={this.context.notifyUpdate}
```